File: proiect_pclp_CATalog.Server/Model/CatModel.cs
namespace proiect_pclp_CATalog.Server.Model
{
    public class CatModel
{
    public int? Id { get; set; }
    public string Name { get; set; }
    public string? Breed { get; set; }
    public int Age { get; set; }
    public string? Sex { get; set; }
    public int Size { get; set; }
    public string? Picture { get; set; }
    public string? VaccinationStatus { get; set; }
    public string? PersonalityTraits { get; set; }
    public bool Adopted { get; set; }
    public string? creationDate { get; set; }
    public string? updatedDate { get; set; }
}
}

File: proiect_pclp_catalog.client/src/app/models/cat.model.ts
import { AppointmentModel } from "./appointment.model";

export interface CatModel {
  id?: number;
  name: string;
  breed: string;
  age: number;
  sex: string;
  size: number;
  picture: string;
  vaccinationStatus: string;
  personalityTraits: string;
  adopted: boolean;
  foster?: string;
  creationDate?: string;
  updatedDate?: string;
  appointment?: AppointmentModel[];
}
